awot/io/read_t28.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import absolute_import, print_function
+import numpy as np
+from netCDF4 import num2date
+from .flight import _winduv
+
+
+def read_t28_netcdf(ncFile):
+    """
+    T-28 has enough differences from other aircraft netCDF files to
+    warrant its own read script.
+
```

Comment:
Did you try the updated NetCDF reader? It may do some of what you want **if** the NetCDF has a global attribute of Conventions with 'nimbus' in it. We could expand this if it works.

One potential gotcha that may pop up is if there is a true 2D variable, then looping through could flatten real 2D data as well.